src/h5web/vis-packs/core/models.ts

```diff
@@ -85,7 +86,9 @@ export type ValueFormatter<T extends DType> = (val: Primitive<T>) => string;
 
 export type ImageAttribute = 'CLASS' | 'IMAGE_SUBCLASS';
 
-export interface HistogramData {
+export interface HistogramParameters {
   data: number[];
   bins: number[];
+  colorMap?: ColorMap;
+  invertColorMap?: boolean;
```

Comment:
The `colorMap` props are enclosed in the `HistogramParameters` as they are only used if there is an histogram and it was easier this way to make the prop go through the `DomainSlider` and `DomainTooltip`